src/components/Search.js

```diff
@@ -1,9 +1,10 @@
 import React from 'react'
 
-const Search = () => {
+const Search = (props) => {
   return (
     <div className="ui huge fluid icon input">
       <input
+        onChange={props.handleChange}
```

Comment:
I would also bind this to the value of the searchTerm for the reasons we discussed in the review